draft-ietf-secevent-subject-identifiers.md

```diff
@@ -265,6 +271,19 @@ Below is a non-normative example Subject Identifier in the Opaque Identifier For
 ~~~
 {: #figexamplesubidopaque title="Example: Subject Identifier in the Opaque Identifier Format"}
 
+### Decentralized Identifier (DID) Format {#sub-id-did}
+The Decentralized Identifier Format identifies a subject using a Decentralized Identifier (DID) URL, identified with a DID URL as defined in {{DID}}.  Subject Identifiers in this format MUST contain a `uri` member whose value is the DID URL for the subject. The value of the `uri` member MAY be any valid DID URL including a bare DID. The `uri` member is REQUIRED and MUST NOT be null or empty. The Decentralized Identifier Format is identified by the name `did`.
+
+Below is a non-normative example Subject Identifier for the Decentralized Identifier Format:
+
+~~~
+{
+  "format": "did",
+  "uri": "did:example:123456"
+}
+~~~
+{: #figexamplesubiddid title="Example: Subject Identifier for the Decentralized Identifier Format"}
+
```

Comment:
Changes `uri` to `url` in accordance with my above suggestion, and adds an example that uses a more complex DID URL.

```suggestion

Below are non-normative example Subject Identifiers for the Decentralized Identifier Format:

~~~
{
  "format": "did",
  "url": "did:example:123456"
}
~~~
{: #figexamplesubiddidbare title="Example: Subject Identifier for the Decentralized Identifier Format, identifying a subject with a bare DID"}

~~~
{
  "format": "did",
  "url": "did:example:123456/did/url/path?versionId=1"
}
~~~
{: #figexamplesubiddidcomplex title="Example: Subject Identifier for the Decentralized Identifier Format, identifying a subject with a DID URL with non-empty path and query components"}
```